map/src/algoliaUtil.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-console */
+import algoliasearch from 'algoliasearch';
+import dotenv from 'dotenv';
+import { v4 as uuidv4 } from 'uuid';
+
+dotenv.config();
+const algoliaAdminKey = process.env.REACT_APP_ALGOLIA_ADMIN_KEY || 'undefined';
+const algoliaAppId = process.env.REACT_APP_ALGOLIA_APP_ID || 'undefined';
+const client = algoliasearch(algoliaAppId, algoliaAdminKey);
```

Comment:
Why are these being defined again if they're in [dataDriver.ts](https://github.com/reach4help/reach4help/blob/map-algolia/map/src/data/dataDriver.ts)? What purpose does this file serve?